### PR TITLE
DOCSP-38547 Standardize Quick Start Name

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-rust/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-38547
Staging - https://preview-mongodbjordansmith721.gatsbyjs.io/rust/DOCSP-38547-quickstart-name/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/quick-start.txt [hunk 1]
```diff
@@ -1,15 +1,22 @@
 .. _rust-quickstart:
 
-===========
-Quick Start
-===========
+================
+Rust Quick Start
```
Comment (ccho-mongodb) [inline source/quick-start.txt:4]: Suggestion:

While the example in the ticket description uses "Go Quick Start" as an example, it also mentions using the product name. In this case, I think "{+driver-long+} Quick Start" might be the most helpful title to understand the content immediately from the results of a Google search.

**Review by ccho-mongodb — APPROVED** [review]
LGTM! Added a suggestion, but maybe @sarahemlin can clarify the ticket request.

File: source/quick-start.txt (hunk 1 above)
Comment (jordan-smith721) [reply source/quick-start.txt:4]: @sarahemlin could you clarify what you're looking for here? For more context "{+driver-long+} Quick Start" would translate to "MongoDB Rust Driver Quick Start" (and similar for the other drivers)

Comment (sarahemlin) [reply source/quick-start.txt:4]: Hey @jordan-smith721! That works for me. The domain kind of takes care of linking this driver to MongoDB, so I would usually put the company name in the meta description. However, it isn't that big of a deal if you all prefer to have the full name in the title. So long as the TOC remains short, I'm okay.

Comment (jordan-smith721) [reply source/quick-start.txt]: Thanks! I discussed it with Chris a bit more over Slack and we decided to go with "Rust Driver Quick Start" as a good middle ground

**Pull request merged**